src/main/java/com/example/bcsd/dto/ArticleResponseDto.java

```diff
@@ -0,0 +1,4 @@
+package com.example.bcsd.dto;
+
+public record ArticleResponseDto(Long id, Long authorId, Long boardId, String title, String content, String createdAt, String modifiedDate) {
```

Comment:
개행을 통해 가독성을 높일 수 있을 것 같아요! 
```suggestion
public record ArticleResponseDto(
    Long id, 
    Long authorId, 
    Long boardId, 
    String title, 
    String content, 
    String createdAt, 
    String modifiedDate
) {
```

Reply:
그러게요 이건 좀 하는게 좋겠네요